### PR TITLE
Channel upgrade

### PR DESCRIPTION
**Pull request by siburu**

### CODE REVIEW

**Comment by bluele** [comment]
memo: we should update the module versions in README before merging.

File: Makefile [hunk 1]
```diff
@@ -142,7 +142,7 @@ enclave:
 .PHONY: clean
 clean:
 	@rm -f $(RustEnclave_Name) $(Signed_RustEnclave_Name) enclave/*_t.* lib/*.a
-	@cd enclave && cargo clean && rm -f Cargo.lock
```
Comment (bluele) [inline Makefile:145]: 👍

**Comment by siburu** [comment]
> memo: we should update the module versions in README before merging.

@bluele I've done this in https://github.com/datachainlab/cosmos-ethereum-ibc-lcp/pull/45/commits/6ab0bf338da02b6d485d325ade63b3817bc688f0

**Review by bluele — APPROVED** [review]
LGTM👍

**Pull request merged**